environment-pipelines/iam.tf

```diff
@@ -832,6 +832,25 @@ resource "aws_iam_policy" "codepipeline" {
   policy      = data.aws_iam_policy_document.codepipeline.json
 }
 
+data "aws_iam_policy_document" "cloudfront" {
+  statement {
+    actions = [
+      "cloudfront:ListCachePolicies",
+      "cloudfront:GetCachePolicy"
+    ]
+    resources = [
+      "arn:aws:cloudfront::${data.aws_caller_identity.current.account_id}:cache-policy/*"
+    ]
+  }
+}
+
+resource "aws_iam_policy" "cloudfront" {
+  name        = "${var.application}-${var.pipeline_name}-pipeline-cloudfront"
+  path        = "/${var.application}/codebuild/"
+  description = "Allow ${var.application} codebuild job access to cloudfront cache policies"
+  policy      = data.aws_iam_policy_document.cloudfront.json
+}
+
```

Comment:
I think `resource "aws_iam_role" "environment_pipeline_codebuild"` still needs to be updated with the new policy - `aws_iam_policy.cloudfront`